lib/winston-mongodb.js

```diff
@@ -107,7 +107,10 @@ let MongoDB = exports.MongoDB = function(options) {
       {strict: false},
       self.capped ? {capped: true, size: self.cappedSize, max: self.cappedMax} : {}
     );
-    return db.createCollection(self.collection, opts).then(col=>{
+    return db.createCollection(self.collection, opts).catch(err=>
+      // swallow connection exists errors
```

Comment:
Nitpick: Should probably say `collection` over `connection`

Reply:
oh lol thanks!!